.github/workflows/test.yml

```
@@ -0,0 +1,73 @@
+  contents: read
+
+jobs:
+  test:
+    name: Test (${{ matrix.os }}, Go ${{ matrix.go-version }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        go-version: ['1.21', '1.22', '1.23']
+      fail-fast: false
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)